### PR TITLE
Show user

### PR DESCRIPTION
**Pull request by NguyenThiHangB**

### CODE REVIEW

**Comment by NguyenThiHangB** [comment]
ready 
Link task: https://dev.framgia.com/issues/184231

File: app/models/book.rb [hunk 1]
```diff
@@ -23,6 +23,13 @@ class Book < ApplicationRecord
   scope :category_id, -> category_id {where category_id: category_id}
   scope :order_by, ->{order created_at: :desc}
 
+  scope :read_by, ->user{joins(:mark_books).where("mark_books.user_id = ? AND
+    mark_books.status = ?", user.id,
+    MarkBook.statuses[:reading]).order created_at: :desc}
+
```
Comment (mdmahmudurrahman) [inline app/models/book.rb]: Remove this empty line, add all scope together

File: app/views/admin/books/_form.html.erb [hunk 2]
```diff
@@ -27,11 +27,7 @@
   </div>
   <div class="form-group">
     <%= f.label :author_id %>
-    <%= f.fields_for :author_books do |ff| %>
-      <%= ff.label :author_id %>
-      <%= ff.select :author_id, options_from_collection_for_select(Author.all,
-        "id", "name"), {include_blank: true}, {class: "chosen-select"} %>
-    <% end %>
+    <%= f.collection_check_boxes :author_ids, Author.all, :id, :name %>
```
Comment (mdmahmudurrahman) [inline app/views/admin/books/_form.html.erb]: Do you need all of the attribute of author here? And get required information in the controller.

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by NguyenThiHangB** [comment]
ready

File: app/controllers/admin/books_controller.rb [hunk 3]
```diff
@@ -37,4 +59,12 @@ def load_book
     flash[:danger] = t ".danger"
     redirect_to admin_books_path
   end
+
+  def list_categories
+    @categories = Category.all
```
Comment (mdmahmudurrahman) [inline app/controllers/admin/books_controller.rb]:
```ruby
Category.all.select(:id, :title).map{|category| [category.title, category.id]}
```

File: app/controllers/admin/books_controller.rb [hunk 4]
```diff
@@ -37,4 +59,12 @@ def load_book
...
+  end
+
+  def list_authors
+    @authors = Author.all
```
Comment (mdmahmudurrahman) [inline app/controllers/admin/books_controller.rb]:
```ruby
Author.all.select(:id, :name).map{|author| [author.name, author.id]}
```

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by NguyenThiHangB** [comment]
ready

**Pull request merged**